server/controllers/inventory/reports/changes.js

```diff
@@ -113,6 +111,14 @@ function formatKeys(record) {
 function getValue(last, current, key) {
   const result = {};
 
+  if (key === 'tags') {
+    const lastTags = last.tags.map(tag => `"${tag.name}"`).join(',  ');
+    result.from = last.tags.length ? `[${lastTags}]` : '[]';
```

Comment:
I think you may not need this length check.  It will be an empty string by default.
```js
    result.from =  `[${lastTags}]` 
 ```
 Should work fine.

Reply:
Done.  And it works!